src/Action/UpdateAdministrationRoleViewAction.php

```diff
@@ -46,9 +47,16 @@ public function __construct(
 
     public function __invoke(Request $request): Response
     {
-        /** @var AdministrationRoleInterface $administrationRole */
+        /** @var AdministrationRoleInterface|null $administrationRole */
         $administrationRole = $this->administrationRoleRepository->find($request->attributes->get('id'));
 
+        if (null === $administrationRole) {
+            throw new NotFoundHttpException(sprintf(
```

Comment:
Shouldn't we better be redirected to administration roles index (or to referer) with a proper flash message?